src/did/connect/utils.js

```diff
@@ -103,4 +103,12 @@ export async function findWebSocketListener(
       }
     }
   }
-}
\ No newline at end of file
+}
+
+export async function triggerProtocolHandler(url) {
```

Comment:
hoping to take a stab at adding `JSDoc` to this before you merge but don't let this hold you back

Reply:
Definitely in need of lots of JSDoc attention.  There were 0 comments before so trying to add as we go along.  That function didn't have any comments previously either... was moved from the general `utils.js` since it is specific only to DID Connect and I want to make it possible to have DID Connect be an optional import in the future.